app/resources/app-base.html.template

```diff
@@ -14,7 +14,13 @@
 {% block title %}{{block.super}}{% endblock title %}
 
 {% block logo %}
-  {% if not env.amp %}
+  {% if env.amp %}
+    <div class="header-space">
+      <a href="/global/">
```

Comment:
Use {{env.global_url}} instead of /global/. Note that the actual URL may differ per config e.g., it is /personfinder/global in our production.

Reply:
done.